scripts/submit_finetune_job.py

```diff
@@ -184,6 +184,9 @@ def parse_args(args):
     d['tasks'][0]['envVars'].append({
         'name': 'WANDB_RUN_GROUP', 'value': experiment_group
     })
+    d['tasks'][0]['envVars'].append({
```

Comment:
Note that we have to add a `BEAKER_TOKEN` secret in the workspace.

Reply:
Is is the beaker user token?

Reply:
We've also been running the tulu 3 evals in a shared workspace, can we make the name of the secret customizable? So I can set (somewhere, cli flag or in the config) that my personal beaker token is named (e.g.) JACOBM_BEAKER_TOKEN?

Reply:
> Is is the beaker user token?
Yes, this is the beaker user token. We need to do `beaker secret write BEAKER_TOKEN xxxxx` in the eval workspace

> We've also been running the tulu 3 evals in a shared workspace, can we make the name of the secret customizable? So I can set (somewhere, cli flag or in the config) that my personal beaker token is named (e.g.) JACOBM_BEAKER_TOKEN?

We can only write a `BEAKER_TOKEN` in the eval workspace. It is possible to support multiple users. E.g., we could have 

```
beaker secret write JACOB_BEAKER_TOKEN xxxxx
beaker secret write COSTA_BEAKER_TOKEN xxxxx
beaker secret write HAMISH_BEAKER_TOKEN xxxxx
...
```


and then in runtime we can do something like `'name': 'BEAKER_TOKEN', 'secret': "$(whoami)$_BEAKER_TOKEN"` Would you prefer this solution? It can also generalize to HF_TOKEN, WANDB_API_KEY.

Reply:
Alright I will prepare some commands for people to run :D

Reply:
I'm personally not a big fan of `"$(whoami)$_BEAKER_TOKEN"` because my username varies on different machines (laptop, ai2 servers, uw machines) and this adds some management. I feel this is ideally something that beaker should be responsible for, or otherwise can we have some non-user specific default somehow? (e.g. service account credential or something)